server/priv/marketing/blog/2025/11/03/website-redesign.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Designing the new marketing site"
+category: "product"
+tags: ["product"] 
+excerpt: "Unveiling the new Tuist website, and a behind-the-scenes look at our evolution from a CLI-only tool to an all-in-one open-source platform for iOS app development."
+author: asmitbm
+og_image_path: /marketing/images/blog/2025/11/03/website-redesign/og.png
+highlighted: true
+---
+
+As you would all guess, Tuist’s main way of communicating is through the website. It’s where developers begin exploring the product, learning about its features, and discovering what Tuist is all about. We wanted to make this experience simple and welcoming, not just visually, but also in how the brand and content feel.
+
+As the product evolved, the old website no longer represented what we’re building and shipping today. After [redesigning our developer dashboard](https://tuist.dev/blog/2025/04/17/meet-new-tuist), we wanted our landing page to match its new look and feel. We began by reviewing what worked well on the previous site and what could be improved, drawing inspiration from other developer tools. We also realized some important information was missing, and with new features launching every cycle, it felt like the perfect time for a redesign.
+
+![Home page of the new marketing site](/marketing/images/blog/2025/11/03/website-redesign/home-page.png)
+
+## The process
+
+Before redesigning the website, we spent time researching what could be improved and what was missing. We reviewed over 25 developer tool websites, studying their strengths and weaknesses. The website plays a big role in shaping our brand identity, and with the dashboard redesign, we wanted to bring the same look and feel to our site.
+
+![Website moodboard](/marketing/images/blog/2025/11/03/website-redesign/moodboard.png)
+
+The main issues we identified were:
+* The old website didn’t clearly communicate that Tuist had grown beyond the CLI.
+* The previous site layout made it hard to find information or understand Tuist’s offering.
+* It was difficult to showcase new features and updates effectively.
+* We weren’t effectively showing how Tuist helps real teams or encouraging new contributors.
+* The website visuals didn’t align with the newly redesigned dashboard.
+
+### Stage 1: Clarifying the Message
+
+The first thing we noticed was that our story wasn’t being told the way we wanted it to be. Tuist had grown far beyond being just a CLI tool, yet our website still gave that impression. We wanted to communicate what Tuist truly is today, a platform that helps teams build, collaborate, and ship iOS apps faster.
```

Comment:
```suggestion
The first thing we noticed was that our story wasn’t being told the way we wanted it to be. Tuist had grown far beyond being just a CLI tool, yet our website still gave that impression. We wanted to communicate what Tuist truly is today, a platform that helps teams build, collaborate, and ship apps.
```

I'm somewhat allergic to the "... apps faster." I'd personally not use that messaging anymore